### PR TITLE
Replace Flask by Quart

### PR DESCRIPTION
**Pull request by ebiiii**
tests are broken, it seems it is not possible to use responses and asyncio simultaneously

### CODE REVIEW

**Comment by rokroskar** [comment]
https://github.com/pnuckowski/aioresponses?

**Comment by ebiiii** [comment]
If we switch to this one we would also have to replace all the "requests" usages with "aiohttp". Maybe later. Here this issue is that tests are run asynchonously and the configuration changes between tests, I need to make them synchonous.

**Comment by leafty** [comment]
You can't make a new event loop per test?

**Comment by ebiiii** [comment]
Yes, this is the way to do it.

File: Dockerfile [hunk 1]
```diff
@@ -21,6 +21,6 @@ COPY endpoints.json .
 #       Currently, the poor mans approach to session handling and the
 #       signaling through blinker do only work within one process!
 
-CMD ["gunicorn", "-b 0.0.0.0:5000", "run:app.app",  "-k gevent"]
+CMD ["hypercorn", "-b0.0.0.0:5000", "run:app.app"]
```
Comment (leafty) [inline Dockerfile]: :unicorn: with multiple horns? Parallel thrusting ftw :rainbow:

File: Pipfile [hunk 2]
```diff
@@ -4,16 +4,18 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-greenlet = "*"
+Werkzeug = "*"
+hypercorn = "*"
 oic = "*"
 requests = "*"
+pytest = "*"
```
Comment (ableuler) [inline Pipfile]: I think these 3 packages could go to dev-packages.

File: helm-chart/renku-gateway/Chart.yaml [hunk 3]
```diff
@@ -3,4 +3,4 @@ appVersion: '1.0'
 description: A Helm chart for the Renku gateway
 name: renku-gateway
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.1.0-a4fb826
+version: 0.0-unclean-cdf747b
```
Comment (ableuler) [inline helm-chart/renku-gateway/Chart.yaml]: The chartpress relics shouldn't be committed - please revert to `version: 0.1.0` while you're at it...

**Comment by ableuler** [comment]
@ebiiii I'd say we merge this to master now that the 0.2.0 release has been tagged?

**Pull request merged**